ush/python/pygfs/utils/marine_da_utils.py

```diff
@@ -0,0 +1,100 @@
+import f90nml
+import os
+from logging import getLogger
+import xarray as xr
+
+from wxflow import FileHandler, logit, WorkflowException, AttrDict, parse_j2yaml, jinja
+
+logger = getLogger(__name__.split('.')[-1])
+
+@logit(logger)
+def run(exec_cmd):
```

Comment:
Not immediately clear to me why this is needed, wxflow has an Execute (or Executable or something like that)

Reply:
exec_cmd is an instance of Executable, run is just a simple wrapper for error trapping and logging. I'll have a look at what's available in Executable.